README.md

```diff
@@ -203,31 +164,20 @@ Example:
 
 ## 👥 Authors <a name="authors"></a>
 
```

Comment:
- [ ] Why dont you change the `Authors` to `author` since you are the only contributor to this project? :smile: